<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>计算器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="5" colspan="4">
       <widget class="QCheckBox" name="checkBox_4">
        <property name="text">
         <string>6个数字</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="14">
       <widget class="QTreeView" name="treeView"/>
      </item>
      <item row="6" column="11">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>4个数字</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>3个数字</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="10" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>88</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="checkBox_3">
        <property name="text">
         <string>5个数字</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>结果：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="12" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>计算从左到右，不按照四则运算顺序</string>
        </property>
       </widget>
      </item>
      <item row="0" column="13">
       <widget class="QPushButton" name="pushButton_2">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3" colspan="6">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="3" colspan="7">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="13">
       <widget class="QPushButton" name="pushButton_5">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
      <item row="4" column="10">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>指定相同列数量：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="12">
       <widget class="QPushButton" name="pushButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>开始计算</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="13">
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>还原</string>
        </property>
       </widget>
      </item>
      <item row="6" column="10">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>去掉并存</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="14">
       <widget class="QListView" name="listView"/>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="9">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>挑选包括</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="12">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>查找</string>
        </property>
       </widget>
      </item>
      <item row="4" column="12">
       <widget class="QPushButton" name="pushButton_4">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>求交集</string>
        </property>
       </widget>
      </item>
      <item row="4" column="11">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>最大值：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5" colspan="5">
       <widget class="QLineEdit" name="lineEdit">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
